ModularTegustation/tegu_items/gadgets/unpowered.dm

```diff
@@ -62,14 +62,18 @@
 	var/commandtype = 1
 	var/commanddelay = 1.5 SECONDS
 	var/cooldown = 0
-	var/static/list/commandtypes = typecacheof(list(
-		/obj/effect/temp_visual/commandMove,
-		/obj/effect/temp_visual/commandWarn,
-		/obj/effect/temp_visual/commandGaurd,
-		/obj/effect/temp_visual/commandHeal,
-		/obj/effect/temp_visual/commandFightA,
-		/obj/effect/temp_visual/commandFightB
-		))
+	var/max_commands = 5
```

Comment:
While I can see the intent here, I am not sure if it's necessary.

Reply:
Im anticipating managers filling the hallways with holo effects during apocalypse bird to light the way.